apps/condo/domains/organization/tasks/syncSbbolBankAccounts.js

```diff
@@ -0,0 +1,11 @@
+const { createCronTask } = require('@open-condo/keystone/tasks')
+const { syncBankAccounts } = require('@condo/domains/organization/integrations/sbbol/sync/syncClientAccounts')
+
+/**
+ * Syncs new bank accounts
+ */
+const syncSbbolBankAccounts = createCronTask('syncSbbolBankAccounts', '0 0 * * *', async (date) => {
+    await syncBankAccounts(date)
```

Comment:
Why you need date here? Is't it redundant?

```suggestion
    await syncBankAccounts()
```

Reply:
Yes it is redundant